doc/tutorials/cloud/01_import_data.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
<p>Now it's time to import our data. We call the <code>IMPORT_PATH</code> script, providing the location of parquet files, their format, the s3 endpoint (which has to match the bucket's configuration) and the name of our connection object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>fixed, thanks!</p>